_config.yml

```diff
@@ -58,7 +58,6 @@ google_cse_token:
 # git-wiki includes some internal themes that you can choose
 # check _layouts folder
 #
-markdown: GFM
```

Comment:
no

Reply:
[The Jekyll docs](https://jekyllrb.com/docs/configuration/markdown/#kramdown-processor) mention that Kramdown and GFM are used by default and there is no need to specify it. What exactly is the problem with this?

Reply:
We've had issues in the past. But I am not familiar with it ngl.

But as my request below, this should be fixed in the other repo to then be cherrypicked here